samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/delivery/ItemContentsBean.java

```diff
@@ -1736,5 +1740,125 @@ public boolean isCancelled() {
 	public boolean isCancellable() {
 		return ItemCancellationUtil.isCancellable(this.itemData);
 	}
+	
+	public boolean isTimedQuestion() {
+		String value = itemData.getItemMetaDataByLabel(ItemMetaDataIfc.TIMED);
+		return (StringUtils.isNotBlank(value) && !value.equalsIgnoreCase("false"));
```

Comment:
!StringUtils.equalsIgnoreCase(Boolean.FALSE.toString(), value) ?

not important, please BE SAFE :D

Reply:
done